bundles/binding/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/HueGenericBindingProvider.java

```diff
@@ -84,10 +84,12 @@ public void processBindingConfiguration(String context, Item item,
 
 		super.processBindingConfiguration(context, item, bindingConfig);
 
-		try {
-
-			if (bindingConfig != null) {
+    if( bindingConfig == null || bindingConfig.length() < 1 ) {
+        logger.warn("No settings configured for Hue binding for item '"+item+"'.");
+        return;
+    }
```

Comment:
The code formatting is configured in [the IDE](https://github.com/openhab/openhab-distro/blob/master/docs/sources/development/ide.md), and it would be great if you were able to set that up on your side.  (Thanks a lot, Chris, for your contributions.)